test/js-corpus.go

```diff
@@ -0,0 +1,154 @@
+	"https://github.com/sparanoid/lightense-images.git",
+	"https://github.com/javve/list.js.git",
+	"https://github.com/meandmax/lory.git",
+	"https://github.com/mahdif/loud-links.git",
+	"https://github.com/julmot/mark.js.git",
+	"https://github.com/yabwe/medium-editor.git",
+	"https://github.com/asvd/microlight.git",
+	"https://github.com/henriquea/minigrid.git",
+	"https://github.com/legomushroom/mojs.git",
+	"https://github.com/jacoborus/nanobar.git",
+	"https://github.com/leongersen/noUiSlider.git",
+	"https://github.com/wagerfield/parallax.git",
+	"https://github.com/VincentGarreau/particles.js.git",
+	"https://github.com/dimsemenov/PhotoSwipe.git",
+	"https://github.com/dbushell/Pikaday.git",
+	"https://github.com/pixijs/pixi.js.git",
+	"https://github.com/algolia/places.git",
+	"https://github.com/selz/plyr.git",
+	"https://github.com/metafloor/bwip-js.git",
+	"https://github.com/c3js/c3.git",
+	"https://github.com/chartjs/Chart.js.git",
+	"https://github.com/gionkunz/chartist-js.git",
+	"https://github.com/nosir/cleave.js.git",
+	"https://github.com/NeXTs/Clusterize.js.git",
+	"https://github.com/GetmeUK/ContentTools.git",
+	"https://github.com/dixso/custombox.git",
+	"https://github.com/date-fns/date-fns.git",
+	"https://github.com/frend/frend.co.git",
+	"https://github.com/bevacqua/dragula.git",
+	"https://github.com/imgix/drift.git",
+	"https://github.com/enyo/dropzone.git",
+	"https://github.com/michaelvillar/dynamics.js.git",
+	"https://github.com/tholman/elevator.js.git",
+	"https://github.com/ritz078/embed.js.git",
+	"https://github.com/eqcss/eqcss.git",
+	"https://github.com/kangax/fabric.js.git",
+	"https://github.com/viljamis/feature.js.git",
+	"https://github.com/chmln/flatpickr.git",
+	"https://github.com/grafijs/grafi.git",
+	"https://github.com/greensock/GreenSock-JS.git",
+	"https://github.com/WickyNilliams/headroom.js.git",
+	"https://github.com/Haroenv/holmes.git",
+	"https://github.com/bevacqua/horsey.git",
+	"https://github.com/bitshadow/iconate.git",
+	"https://github.com/impress/impress.js.git",
+	"https://github.com/tholman/intense-images.git",
+	"https://github.com/juliangarnier/anime.git",
+	"https://github.com/googlecreativelab/anypixel.git",
+	"https://github.com/michalsnik/aos.git",
+	"https://github.com/algolia/autocomplete.js.git",
+	"https://github.com/LeaVerou/awesomplete.git",
+	"https://github.com/kennethcachia/Background-Check.git",
+	"https://github.com/feimosi/baguetteBox.js.git",
+	"https://github.com/kazzkiq/balloon.css.git",
+	"https://github.com/rishabhp/bideo.js.git",
+	"https://github.com/dinbror/blazy.git",
+	"https://github.com/tictail/bounce.js.git",
+	"https://github.com/callmecavs/bricks.js.git",
+}
+
+func git_clone_url(url string, wg *sync.WaitGroup) {
+	defer fmt.Println("(cloned)", url)
+	cmd := exec.Command("git", "clone", "--depth", "1", url)
+	cmd.Stdout = nil
+
+	if err := cmd.Run(); err != nil {
+		fmt.Println("(failed)", url)
+	}
+}
+
+func main() {
```

Comment:
I think it's a bit unexpected that this script dumps a bunch of folders into the current directory. The program should require a command-line argument specifying the parent directory in which to put the corpus.